utils/src/lib.rs

```diff
@@ -225,7 +225,22 @@ pub fn link_with_nonce_or_address<T, U>(
                 }
                 BytecodeObject::Bytecode(ref bytes) => {
                     if bytes.as_ref().is_empty() {
-                        // abstract, skip
+                        // Handle case where bytecode bytes are empty
```

Comment:
@mattsse This is how I am currently getting Interfaces to be included in the `known_contracts`. Might add a bit of over overhead as each Abstract contract and interface's abi will be kept in memory. Any objections to this approach?

Reply:
I think this is fine for now—we can profile and optimize later if it slows things down.